samtranslator/model/apigatewayv2.py

```diff
@@ -57,39 +58,160 @@ class ApiGatewayV2ApiMapping(Resource):
 
 class ApiGatewayV2Authorizer(object):
     def __init__(
-        self, api_logical_id=None, name=None, authorization_scopes=[], jwt_configuration={}, id_source=None,
+        self,
+        api_logical_id=None,
+        name=None,
+        authorization_scopes=[],
+        jwt_configuration={},
+        id_source=None,
+        function_arn=None,
+        function_invoke_role=None,
+        identity=None,
+        authorizer_payload_format_version=None,
+        enable_simple_responses=None,
     ):
         """
         Creates an authorizer for use in V2 Http Apis
         """
-        if authorization_scopes is not None and not isinstance(authorization_scopes, list):
```

Comment:
Why we delete this vaildation, and the validations below?

Reply:
I've just moved it further down, after we initialize all variables